README.md

```diff
@@ -5,6 +5,26 @@ with Ruby on Rails applications.
 
 ## Usage
 
+**At the moment, we've confirmed that the fonts and stylesheets in this repo
+work in Rails apps.  JavaScript is TBD.**
+
+Until this gem is released, you can load it via GitHub:
+
+```
+gem 'uswds-rails', github: '18F/uswds-rails-gem'
+```
+
+Then bundle and include the stylesheets in your manifest in this order:
+
+```
+# app/assets/stylesheets/application.scss
+
+@import 'uswds/all';
+@import 'uswds_overrides/all';
```

Comment:
wonder if we should rename this dir to something more obvious like `uswds_rails_overrides` ...